staticaddr/deposit/fsm.go

```diff
@@ -203,7 +211,7 @@ func NewFSM(ctx context.Context, deposit *Deposit, cfg *ManagerConfig,
 				return
 			}
 		}
-	}()
+	}(depoFsm)
```

Comment:
Why is this passing-as-an-argument needed? Previous version seems also ok.

Reply:
usually this pattern is better within loops, but I also find in more clear, so keeping it.